### DISCUSSION TITLE
Clipboard Access to Host Denial

### DISCUSSION THREAD

**Discussion by HqT65W6R** [sandboxie-plus/Sandboxie#4147] [Various ideas and suggestions]
Clipboard is system-wide and I think it is a good idea for Sandboxie hide clipboard from host when performing copy/paste activities within sandboxes.

**Comment by wilders-soccerfan** [comment]
Please search for OpenClipboard in Sandboxie's [changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md)

**Comment by love-code-yeyixiao** [comment]
Clipboard is window-station-wide.






---Original---
From: ***@***.***&gt;
Date: Sun, Aug 11, 2024 01:23 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [sandboxie-plus/Sandboxie] Clipboard Access to Host Denial(Discussion #4147)




 
Clipboard is system-wide and I think it is a good idea for Sandboxie hide clipboard from host when performing copy/paste activities within sandboxes.
